src/components/loader-spinner/loader-spinner.test.tsx

```diff
@@ -223,7 +219,7 @@ describe("when custom props are passed", () => {
     const hiddenLabelElement = screen.getByTestId("hidden-label");
 
     expect(wrapperElement).toHaveTextContent("Loading...");
-    expect(hiddenLabelElement).toBeInTheDocument();
+    expect(hiddenLabelElement).not.toBeVisible();
```

Comment:
comment: I think it's worth keeping the assertion as before but adding the new one etc. That way we're asserting that the label is there but not visible etc
```
expect(hiddenLabelElement).not.toBeVisible();
expect(hiddenLabelElement).toBeInTheDocument();

Reply:
same comment about asserting the visible label is not in the document instead of the hidden label not being visible